bin/testrun

```diff
@@ -22,11 +22,16 @@ fi
 # Ensure that /var/run/netns folder exists
 sudo mkdir -p /var/run/netns
 
-export TESTRUNPATH=/usr/local/testrun
-cd $TESTRUNPATH
+FILE_PATH=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-# Create device folder if it doesn't exist
-mkdir -p local/devices
+# Check if Testrun was installed as a package
+if [ $FILE_PATH == "/usr/bin" ] ; then
+  export TESTRUN_PATH="/usr/local/testrun"
+else
+  export TESTRUN_PATH=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/..
```

Comment:
This doesn't resolve the root testrun directory as expected, should be updated to:

`export TESTRUN_PATH=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")"/.. && pwd)`

Reply:
Does this not work for you? It works for me currently

Reply:
How are you running testrun?  I run it from the testrun folder with the command sudo bin/testrun which resolves the TESTRUN_PATH incorrectly but the updated code works as expected.

Reply:
sudo bin/testrun is working for me after local install

Reply:
Both methods seem to work. I have used your suggestion in case that solves any issues on your end.